manifests/configfile.pp

```diff
@@ -64,6 +65,7 @@
 
   if($config) {
     file { $config_file:
+      ensure  => $ensure,
```

Comment:
```suggestion
      ensure  => $logstash::ensure,
```

Reply:
if there's a `$ensure` parameter, we should also use it?